server/routes/visitors.test.ts

```diff
@@ -697,14 +697,19 @@ describe('/visit/select-date-and-time', () => {
   })
 
   describe('POST /visit/select-date-and-time', () => {
+    const visitSessionsService = new VisitSessionsService(null, systemToken) as jest.Mocked<VisitSessionsService>
+
     beforeEach(() => {
-      sessionApp = appWithAllRoutes(null, null, null, null, systemToken, false, {
+      visitSessionsService.createVisit = jest.fn().mockResolvedValue('2a-bc-3d-ef')
```

Comment:
Do we need to do this here if we're not reseting them on an `afterEach` ?

Reply:
Just tried that but it didn't work - they're getting reset on an `afterEach()` right at the top of the file. One to consider when we split up this visits stuff!

Reply:
I think it's all to do with scope, but yeah we can address it all when we split it up 👍